### DISCUSSION TITLE
I'd like to get your opinion.about my server lilac.log

### DISCUSSION THREAD

**Discussion by CatCat-Jar** [J-Tanzanite/Little-Anti-Cheat#124] [Q&A] [answered]
Sorry,my english is not good.

[lilac.log](https://github.com/J-Tanzanite/Little-Anti-Cheat/files/8984113/lilac.log)

https://user-images.githubusercontent.com/106509773/175763522-8950c19b-f675-4c9d-8123-dbb23f479de8.mp4
this is the 14:32:04 line video,He recorded a video of me, and my server has a timeline

I'd like to get your opinion.

**Comment by azalty** [comment]
You got a single detection, what's the matter?

**Reply by CatCat-Jar** [reply]
No, there are three people here.  Is the operation in the video considered to be aimbot.I think it's normal.what do you think?

**Reply by azalty** [reply]
You only show one person in this clip, who is it?

**Reply by CatCat-Jar** [reply]
14:32:04

**Reply by azalty** [reply, marked as answer]
Again, in this video there is a single detection `2022/06/25 14:32:04 [Version 1.7.1] {Name: "2" | SteamID: 2 | IP: 2} is suspected of using an aimbot (Detection: 2 | Delta: 0 | TotalDelta: 0 | Detected: Autoshoot).`

As long as there aren't multiple, you shouldn't worry
